templates/etc/snmp/snmpd.conf.j2

```diff
@@ -31,6 +31,9 @@ includeAllDisks {{ snmpd_disks_include_all_threshold_minpercent }}
 {% for snmpd_disk in snmpd_disks %}
 disk {{ snmpd_disk.path }} {{ snmpd_disk.threshold }}
 {% endfor %}
+{% if ansible_facts['processor_count'] is defined and ansible_facts['processor_cores'] is defined %}
+load {{ ansible_facts['processor_count'] * ansible_facts['processor_cores'] }}
+{% endif %}
```

Comment:
I rather have something like this:

```
{% if snmpd_load is defined %}
load {{ snmpd_load }}
{% endif %}
```

Then you can fill it in the way you want (or have it `undefined`)

Reply:
fair